apps/src/templates/tables/QuickActionsCell.jsx

```diff
@@ -67,6 +75,17 @@ export default class QuickActionsCell extends Component {
     this.updateMenuLocation();
     window.addEventListener("resize", throttle(this.updateMenuLocation, 50));
     this.setState({open: true, canOpen: false});
+    if (this.props.experimentDetails) {
+      firehoseClient.putRecord(
```

Comment:
nit: It would be cleaner if the `experimentDetails` prop matched the shape of the argument to `putRecord`, so that this line could just be `firehoseClient.putRecord(this.props.experimentDetails)`

Reply:
ooh, that would be tidy. How could I do  `data_json: JSON.stringify({ channel_id: this.props.experimentDetails.projectId })` like that though?

Reply:
You would have to move that `JSON.stringify` call to where you pass the prop into `QuickActionsCell`, and replace the `projectId` field in your PropType declaration with a `data_json` one.